src/ui/verificationSymbolStatusView.ts

```diff
@@ -154,36 +136,49 @@ export default class VerificationSymbolStatusView {
 
   private async update(params: IVerificationSymbolStatusParams): Promise<void> {
     await this.noRunCreationInProgress;
+
     const uri = Uri.parse(params.uri);
     params.uri = uri.toString();
     const document = await workspace.openTextDocument(uri);
     const rootSymbols = await commands.executeCommand('vscode.executeDocumentSymbolProvider', uri) as DocumentSymbol[] | undefined;
-
     // After this check we may no longer use awaits, because they allow the document to be updated and then our update becomes outdated.
-    if(params.version !== document.version) {
+    if((params.version ?? 1) !== document.version) {
```

Comment:
Is the first version 1 or 0? '??' also coerces on 0

Reply:
I think `||` also coerces on `0`, but `??` also does so on `undefined`

Reply:
The first version is `1`